libraries/ms-common/inc/crc32.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+// CRC32 module
+// Polynomial 0x04C11DB7, initial value 0xFFFFFFFF - standard CRC32 model
+#include <stddef.h>
+#include <stdint.h>
+#include "status.h"
+
+StatusCode crc32_init(void);
```

Comment:
Why return a `StatusCode` it looks like this can't fail?

Reply:
Consistency? I can remove it I guess